photologue/forms.py

```diff
@@ -18,6 +21,27 @@
 
 logger = logging.getLogger('photologue.forms')
 
+MessageSeverity = int
+MessageContent = str
+
+@dataclass
```

Comment:
dataclasses were introduced in Python 3.7. Django-photologue tries to support the same versions as django - and django 3.2 (still supported) is compatible with Python 3.6. 
So we should avoid using dataclasses.